### PR TITLE
Fix for issue #37

### PR DESCRIPTION
**Pull request by TheAifam5**
That fix allows to select the .NET Framework version in the window where the user can create a projects.

### CODE REVIEW

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismCarouselPage/PrismCarouselPage.vstemplate [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismCarouselPage.ico</Icon>
     <TemplateID>d174f6d6-4267-4dd8-be14-aa4ef6cf4a18</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/Forms/PrismCarouselPage/PrismCarouselPage.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismContentPage/PrismContentPage.vstemplate [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismContentPage.ico</Icon>
     <TemplateID>a361a5e9-0592-4a77-8442-22f54a426a3d</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/Forms/PrismContentPage/PrismContentPage.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismMasterDetailPage/PrismMasterDetailPage.vstemplate [hunk 3]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismMasterDetailPage.ico</Icon>
     <TemplateID>3ca4fa2d-810f-45ae-aa14-69e1cb7f855e</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/Forms/PrismMasterDetailPage/PrismMasterDetailPage.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismNavigationPage/PrismNavigationPage.vstemplate [hunk 4]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismNavigationPage.ico</Icon>
     <TemplateID>57383ffc-2f32-4b59-b6e3-209d869cdeee</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/Forms/PrismNavigationPage/PrismNavigationPage.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismTabbedPage/PrismTabbedPage.vstemplate [hunk 5]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismTabbedPage.ico</Icon>
     <TemplateID>011fad1d-22b2-45f3-95a4-b93035378e2c</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/Forms/PrismTabbedPage/PrismTabbedPage.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/PrismViewModel/PrismViewModel.vstemplate [hunk 6]
```diff
@@ -6,6 +6,7 @@
     <Icon>PrismViewModel.ico</Icon>
     <TemplateID>bfeb7adb-f776-4dfa-8cd3-dc796dbd8533</TemplateID>
     <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
```
Comment (brianlagunas) [inline TemplatePack/ItemTemplates/CSharp/PrismViewModel/PrismViewModel.vstemplate]: We don't need it for the ItemTemplates

File: TemplatePack/ItemTemplates/CSharp/Forms/PrismCarouselPage/PrismCarouselPage.vstemplate (hunk 1 above)
Comment (TheAifam5) [reply TemplatePack/ItemTemplates/CSharp/Forms/PrismCarouselPage/PrismCarouselPage.vstemplate]: I’m gonna fix that in the next 1-2h.

**Comment by TheAifam5** [comment]
I seen that the ItemTemplates like for WPf have already defined the RequiredFrameworkVersion. Should I remove it also?

**Comment by TheAifam5** [comment]
What I am talking about: https://github.com/PrismLibrary/Prism-Extensibility/blob/master/TemplatePack/ItemTemplates/CSharp/Wpf/PrismWindow/PrismWindow.vstemplate

**Comment by brianlagunas** [comment]
No need to remove anything that is already there. Just don't add it if it's not necessary.  Thanks for checking

**Comment by brianlagunas** [comment]
Thanks for the PR

**Pull request merged**